Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -1374,7 +1449,7 @@ def delete_group():
 def get_mapping_fields_command(search_base):
     ad_attributes = get_all_attributes(search_base)
     # add keys that are not attributes but can be used in mapping
-    ad_attributes.extend(("dn", "samaccountname"))
+    ad_attributes.extend(("dn", "samaccountname", "manageremail"))
```

Comment:
why aren't they returned like all the rest of the attributes?

Reply:
It's an old fix we had, those field exists in AD but we couldn't get them with the rest of the attributes.